ajax/libs/react-disqus-thread/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "author": {
+    "name": "Matt Zabriskie"
+  },
```

Comment:
Use `"author": "Matt Zabriskie"` is easier since there is only "name" in it, please try it next time.